### PR TITLE
Fix: Anchor Widget - Enforced better security policies [ED-7250]

### PR DESCRIPTION
**Pull request by matipojo**

### CODE REVIEW

File: tests/phpunit/elementor/includes/widgets/test-menu-anchor.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Elementor\Tests\Phpunit\Includes\Widgets;
+
+use Elementor\Plugin;
+use ElementorEditorTesting\Elementor_Test_Base;
+
+class Elementor_Test_Element_Base extends Elementor_Test_Base {
```
Comment (StyleShit) [inline tests/phpunit/elementor/includes/widgets/test-menu-anchor.php:8]:
```suggestion
class Test_Menu_Anchor extends Elementor_Test_Base {
```

File: tests/phpunit/elementor/includes/widgets/test-menu-anchor.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	const MOCK = [
+		'id' => 'e8e55a1',
+		'elType' => 'widget',
+		'settings' => [ 'anchor' => 'test-anchor' ],
+		'widgetType' => 'menu-anchor',
+	];
+
+	public function test_on_save__with_invalid_data() {
```
Comment (StyleShit) [inline tests/phpunit/elementor/includes/widgets/test-menu-anchor.php:16]: You're actually testing `get_data_for_save()`

Comment (StyleShit) [reply tests/phpunit/elementor/includes/widgets/test-menu-anchor.php:16]:
```suggestion
	public function test_get_data_for_save__with_invalid_data() {
```

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: tests/phpunit/elementor/includes/widgets/test-menu-anchor.php (hunk 2 above)
Comment (matipojo) [reply tests/phpunit/elementor/includes/widgets/test-menu-anchor.php:16]: The method `get_data_for_save` triggers the protected `on_save` method.

**Pull request merged**